### PR TITLE
[PAN-2965] Added sample load balancer configurations 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added sample load balancer configurations

### CODE REVIEW

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+description: Sample Load Balancers 
+<!--- END of page meta data -->
+
+# Sample Load Balancer Configurations 
+
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Change to `The Classic Load Balancer is the easiest to configure and work with`. Remove the layer  & 7 which may complicate things

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
+ 
+* Configure 1 target group with n nodes 
+* Use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) for health checks
+* Register the Besu instances multiple times with different ports. Similar to configuring microservices 
+on Elastic Container Service (ECS) or Elastic Kubernetes Service (EKS) 
+* Configure multiple listeners with one per port you're using and route to the target group.
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Move this bullet point to be the second one. Rename `one per port` to `one per port (eg: 30303, 8545 etc)`

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Elastic Kubernetes Service  
+
+For Elastic Kubernetes Service (AWS Kubernetes service) use the same load balancer configuration as when
+running nodes in Kubernetes. Use labels to specify nodes for the load balanced group. 
+
+## Manual Configurations 
+
+Where applicable, we strongly recommend using service discovery. That is, pair your load balancer 
+configuration with something that dynamically detects new nodes and failures.
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Rename `failures` to `removed failed nodes`

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+# Sample Load Balancer Configurations 
+
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Rename to `Use the Application Load Balancer to get finer grain control.`

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+ 
+* Configure 1 target group with n nodes 
+* Use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) for health checks
+* Register the Besu instances multiple times with different ports. Similar to configuring microservices 
+on Elastic Container Service (ECS) or Elastic Kubernetes Service (EKS) 
+* Configure multiple listeners with one per port you're using and route to the target group. 
+
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Https Redirection:
With either of the AWS load balancers, you can add certificates via ACM (Amazon Certificate Manager) and add them to the load balancers and redirect all http calls to https.

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    backend back-rpc-tcp-8545
+        balance leastconn
+        server node-01 10.0.1.1:8545 weight 1 check
+        server node-02 10.0.1.2:8545 weight 1 check
+        option ....
+        timeout server 600s 
+    ...                
+    ```
+
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Https Redirection:
To add https capability, update the above server blocks to include the certs and specific ciphers. If you require a http to https redirection, add separate blocks to return a 301 code with the new uri

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
+ 
+* Configure 1 target group with n nodes
```
Comment (joshuafernandes) [inline docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Add `You can register each node with the same target group multiple times using different ports, and the listener rules handle routing`

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Done

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Done

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Done

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Done

File: docs/HowTo/Configure/Configure-HA/Sample-Configuration.md (hunk 5 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Configure-HA/Sample-Configuration.md]: Done

**Pull request merged**